requirements.txt

```diff
@@ -1,16 +1,16 @@
-billiard==3.3.0.23
 cairocffi==0.7.2
 cffi==1.7.0
 cryptography==1.4
 gitdb==0.6.4
 MarkupSafe==0.23
 Pillow==3.3.0
+psycopg2==2.6.2; platform_python_implementation=='CPython'
 psycopg2cffi==2.7.4
 pycrypto==2.6.1
 python_axolotl_curve25519==0.1
 PyYAML==3.11
 rcssmin==1.0.6
 rjsmin==1.0.12
 simplejson==3.8.2
-Twisted==16.3.0
+Twisted==16.3.0; python_version<'3'
```

Comment:
Oh, I see the commit message. Is there any real reason *not* to build it on py3? If so, can you add a comment to that effect?